release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1590,15 +1590,41 @@ Previously, the image pruner stopped when it failed to delete an image. As a res
+
+* The `Info` log level used for transient errors caused verbose OLM Operator logs for the default configuration. This fix changes the
+transient error log level to debug. As a result, fewer non-critical logs are visible for the debug configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925614[*BZ#1925614*])
+
+* Previously, the `subscription.spec.config.resources` section was always applied to the installed deployment, even when it was unset or empty. This caused resources defined in the cluster service version (CSV) to be ignored, and only the resources defined in the `subscription.spec.config.resources` section were used. This fix updates OLM to override deployment-specific resources only when the `subscription.spec.config.resources` section is set to a non-nil or non-empty value. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1926893[*BZ#1926893*])
+
+*	During dependency and upgrade resolution, subscription uniqueness was previously based on the subscribed package name. If two subscriptions in a namespace subscribe to the same package, they are treated as a single subscription internally, resulting in unexpected behaviors. With this fix, subscriptions are now uniquely identified internally within a namespace by `.metadata.name` instead of `.spec.name`. As a result, upgrade and dependency resolution behavior is consistent for namespaces containing multiple `Subscription` objects with the same `.spec.name.` (link:https://bugzilla.redhat.com/show_bug.cgi?id=1932001[*BZ#1932001*])
+
+* When less than one minute remains before an upcoming catalog update polling attempt, the interval jitter function truncates the resync interval down to zero. This caused the Operator Catalog to enter a hot-loop, wasting CPU cycles. This fix increases precision of the jitter function used to calculate resync delays. As a result, the Catalog Operator remains mostly idle until the next catalog update poll.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1932182[*BZ#1932182*])
+
+* During an Operator upgrade, the owner reference of any associated `ServiceAccount` objects was updated to point to the new `ClusterServiceVersion` (CSV) object instead of the old one. This could cause a race condition to occur between the OLM Operator, which reconciles CSVs, and the Catalog Operator, executes install plans, marking the old CSV as `Pending/RequirementsNotMet` due to the service account ownership change. This blocked upgrade completion while the new CSV waited indefinitely for the old CSV to indicate a healthy status. With this fix, instead of updating owner references in one step, the second owner is now appended to any existing owners. As a result, the same service account can satisfy requirements for both the old and the new CSV. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1934080[*BZ#1934080*])
```

Comment:
Can/should you s/object/object? I know what you have is grammatically correct; it makes the sentence harder to read (for me anyway) to have _objects_. Assuming _object_ is technically correct, which I *think* it is.